experimental/README.md

```diff
@@ -0,0 +1,20 @@
+# Docker Machine Experimental Features 
+
+This page contains a list of features in the Docker Machine which are
+experimental. Experimental features are **not** ready for production. They are
+provided for test and evaluation in your sandbox environments.  
+
+The information below describes each feature and the Github pull requests and
+issues associated with it. If necessary, links are provided to additional
+documentation on an issue.  As an active Docker user and community member,
+please feel free to provide any feedback on these features you wish.
+
+## Current experimental features
+
+Currently, you can experiment with [migrating from Boot2Docker CLI to Docker Machine](b2d_migration.md). Also, consider reviewing our [rough plan for the migration](b2d_migration_tasks.md). 
```

Comment:
Should this be a list? i.e.: 

"Our current list of experimental features is:

- Migrating boot2docker-cli created VMs to Machine
- Support for Red Hat, Debian, CentOS, Fedora, and RancherOS as base OSes
- etc.
"

Any guidance on which new features should be considered experimental and why?  Generally I think of all new features as fairly experimental for one release, so it seems a bit odd to call out particular ones unless we have a specific area we'd worry about such as confusing users, or the implementation is really hacky but badly needed, etc.